### PR TITLE
FileUploader - Add doc comments for custom upload operations

### PR DESCRIPTION
**Pull request by vladkovl**

### CODE REVIEW

File: js/ui/file_uploader.js [hunk 1]
```diff
@@ -350,8 +351,41 @@ class FileUploader extends Editor {
...
+            * @type_function_return Promise<any>|any
+            */
             uploadFile: null,
+
+            /**
+            * @name dxFileUploaderOptions.uploadChunk
+            * @type function
+            * @type_function_param1 file:File
+            * @type_function_param2 chunksInfo:object
+            * @type_function_param2_field1 bytesLoaded:Number
```
Comment (RomanTsukanov) [inline js/ui/file_uploader.js]: Maybe "bytes**Up**loaded"?

File: js/ui/file_uploader.js [hunk 2]
```diff
@@ -350,8 +351,41 @@ class FileUploader extends Editor {
...
+            * @type_function_param2 progressCallback(loadedBytes):any
+            * @type_function_return Promise<any>|any
+            */
             uploadFile: null,
+
+            /**
+            * @name dxFileUploaderOptions.uploadChunk
+            * @type function
+            * @type_function_param1 file:File
+            * @type_function_param2 chunksInfo:object
```
Comment (RomanTsukanov) [inline js/ui/file_uploader.js]: Looks like an object with information on a single chuck, not on several. Shouldn't it be "chunkInfo" without "s"?

File: js/ui/file_uploader.js (hunk 1 above)
Comment (vladkovl) [reply js/ui/file_uploader.js]: "`bytesUploaded`" is better. But "`bytesLoaded`" has already been used in `onProgress` event.

File: js/ui/file_uploader.js (hunk 2 above)
Comment (vladkovl) [reply js/ui/file_uploader.js]: This members applies to all chunks: `bytesLoaded`, `chunkCount`, `customData`. But I think it should not be a problem if we change the parameter name to `chunkInfo` and at the same time keep all this members inside this object.

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**